### DISCUSSION TITLE
The primer theme is not currently supported on GitHub Pages

### DISCUSSION THREAD

**Discussion by sayanee** [community/community#23365] [Other Feature Feedback, Questions, & Ideas]
<p>The latest build for my GitHub pages is failing with the error message “The primer theme is not currently supported on GitHub Pages.”</p>
<p>This is <a href="https://github.com/hutscape/oak" rel="noopener nofollow ugc">not a new repository</a>. It has <a href="https://hutscape.com/oak/" rel="noopener nofollow ugc">a published GitHub pages website</a>. The last commit that failed the build did not make any changes related to GitHub pages.</p>
<p>I have already tried re-running the build and also committing a new change, but the build failure error message is still the same.</p>
<p>Thanks for your help!</p>

**Comment by dangowans** [comment]
<p>You’re not alone.  Looks like the cayman theme is having the same issue.</p>
<aside class="onebox githubissue" data-onebox-src="https://github.com/pages-themes/cayman/issues/130">
  <header class="source">

      <a href="https://github.com/pages-themes/cayman/issues/130" target="_blank" rel="noopener nofollow ugc">github.com/pages-themes/cayman</a>
  </header>

  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Issue">
	  <svg width="60" height="60" class="github-icon" viewbox="0 0 14 16" aria-hidden="true"><path d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"></path></svg>
  </div>

  <div class="github-info-container">
    <h4>
      <a href="https://github.com/pages-themes/cayman/issues/130" target="_blank" rel="noopener nofollow ugc">The cayman theme is not currently supported on GitHub Pages</a>
    </h4>

    <div class="github-info">
      <div class="date">
        opened <span class="discourse-local-date" data-format="ll" data-date="2021-07-23" data-time="07:57:14" data-timezone="UTC">07:57AM - 23 Jul 21 UTC</span>
      </div>


      <div class="user">
        <a href="https://github.com/DSoko2" target="_blank" rel="noopener nofollow ugc">
          <img alt="DSoko2" src="https://user-images.githubusercontent.com/19495149/181076555-7b8bd182-c102-4e24-a892-0cb44f136be9.jpeg" class="onebox-avatar-inline" width="20" height="20">
          DSoko2
        </a>
      </div>
    </div>

    <div class="labels">
    </div>
  </div>
</div>

  <div class="github-row">
    <p class="github-body-container">I get the following error since July 22nd 2021 when building Github Pages with t<span class="show-more-container"><a href="" rel="noopener" class="show-more">…</a></span><span class="excerpt hidden">he cayman theme, if the following two lines are included in the _config:

```
github:
  is_project_page: false
```

Removing these from the _config solves the issue in my case. However, I would like to have that configuration to hide the "View on GitHub" button.

The build error:

```
The page build failed for the `main` branch with the following error:

The cayman theme is not currently supported on GitHub Pages. For more information, see https://docs.github.com/github/working-with-github-pages/adding-a-theme-to-your-github-pages-site-using-jekyll.
```

Repo I investigated and faced this issue on: https://github.com/DSoko2/Corona-NI</span></p>
  </div>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by AnonyPla** [comment]
<p>Indeed you’re not alone, same issue with the hacker theme since yesterday. I opened a support ticket and they’re looking into it. I mentioned it was also affecting other themes.</p>
<p><a href="https://github.community/t/the-hacker-theme-is-not-currently-supported-on-github-pages/191708">https://github.community/t/the-hacker-theme-is-not-currently-supported-on-github-pages/191708</a></p>
<aside class="onebox githubissue" data-onebox-src="https://github.com/pages-themes/hacker/issues/77">
  <header class="source">

      <a href="https://github.com/pages-themes/hacker/issues/77" target="_blank" rel="noopener nofollow ugc">github.com/pages-themes/hacker</a>
  </header>

  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Issue">
	  <svg width="60" height="60" class="github-icon" viewbox="0 0 14 16" aria-hidden="true"><path d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"></path></svg>
  </div>

  <div class="github-info-container">
    <h4>
      <a href="https://github.com/pages-themes/hacker/issues/77" target="_blank" rel="noopener nofollow ugc">The hacker theme is not currently supported on GitHub Pages.</a>
    </h4>

    <div class="github-info">
      <div class="date">
        opened <span class="discourse-local-date" data-format="ll" data-date="2021-07-23" data-time="15:56:14" data-timezone="UTC">03:56PM - 23 Jul 21 UTC</span>
      </div>


      <div class="user">
        <a href="https://github.com/AnonyPla" target="_blank" rel="noopener nofollow ugc">
          <img alt="AnonyPla" src="https://user-images.githubusercontent.com/86740652/181076557-f50ef2c0-a8f7-41c9-97e2-6f368cad4597.jpeg" class="onebox-avatar-inline" width="20" height="20">
          AnonyPla
        </a>
      </div>
    </div>

    <div class="labels">
    </div>
  </div>
</div>

  <div class="github-row">
    <p class="github-body-container">Since yesterday (07/22/2021) I constantly get the following error:

" The page<span class="show-more-container"><a href="" rel="noopener" class="show-more">…</a></span><span class="excerpt hidden"> build failed for the main branch with the following error: The hacker theme is not currently supported on GitHub Pages. "

No weird changes were made and it was working fine the day before. 

![Capture](https://user-images.githubusercontent.com/86740652/126808741-087465b7-e122-4753-9096-4c40fa30bb3d.jpg)

![Capture2](https://user-images.githubusercontent.com/86740652/126808760-6201ed89-15db-4236-8182-05ff624a461c.jpg)</span></p>
  </div>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by parkr** [comment, marked as answer]
<p><a class="mention" href="/sayanee">@sayanee</a> First of all, that is such a cool project! I’m sorry you’re having trouble building your site today. Would you do me a favor and try building again?</p>

**Comment by sayanee** [comment]
<p>Thanks, <a class="mention" href="/parkr">@parkr</a>! I just re-ran the last commit build and it has passed. I guess it was an intermittent issue for a few hours.</p>

**Comment by Jmuccigr** [comment]
<p>See similar reports <a href="https://github.community/t/jekyll-theme-minimal-not-working/191791/">here</a> for the minimal and hacker themes. Still not working for me.</p>

**Comment by filipvanlaenen** [comment]
<p>I think there is a general problem with the themes. I made a change to the <code>assets/css/style.scss</code> file on a site with the Cayman theme, and it’s throwing the following error at me even though it complies with the instructions on how to update that file:</p>
<blockquote>
<p>Your SCSS file <code>assets/css/style.scss</code> has an error on line 1: File to import not found or unreadable: . Load paths: _sass /tmp/jekyll-remote-theme-20210724-6-1i1pm44/_sass.</p>
</blockquote>
<p>If I remove the file, it works. But if I add it again to make a custom change to the theme, it’s broken. It breaks on the import statement, whether I use <code>""</code> or <code>"{{ site.theme }}"</code>.</p>
